### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader: NVDA is not announcing the label for the non editable text box,edit box and drop down combo box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6154]
**"Bug Description:**
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS;  #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details**:
**Application name**: Azure Data Studio (Agent, Create External Data, Notebooks)
App Version: 1.9.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details:
**Tool Name**: NVDA
**MAS Reference**:MAS1.3.1
**Repro Steps**:
1.Install the app Azure Data Studio.
2.Enable SQL server2019 preview.
3.Select table from the database.
4.Create External Table screen should open->Next button.
5.Create Database Master Key screen should open->click on next button.
6.Fill all the required details->click on next button.
7.Open NVDA and Observe the announcement. 
**Actual**:
Screen Reader is not announcing the level for the non editable text box ,edit box, drop down combo box. It is announcing the selected text inside the text box.
**Expected**:
It should announce the level for non editable text box, edit box, drop down combo box.
**User Impact**:
User with disabilities will not be  able to relate with the edit boxes that what input he or she needs to give in the related box.
**Recommendation**:
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference**:MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference**:
[6154_A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3324903/6154_A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx)
